native/ssz_nif/src/utils/from_lh.rs

```diff
@@ -60,3 +63,67 @@ where
         v.into_iter().map(|x| Elx::from(x, env)).collect::<Vec<_>>()
     }
 }
+
+impl<'a> FromLH<'a, ExecutionBlockHash> for Binary<'a> {
+    fn from(value: ExecutionBlockHash, env: rustler::Env<'a>) -> Self {
+        bytes_to_binary(env, &value.as_ssz_bytes())
+    }
+}
+
+impl<'a> FromLH<'a, H160> for Binary<'a> {
+    fn from(value: H160, env: rustler::Env<'a>) -> Self {
+        bytes_to_binary(env, &value.as_ssz_bytes())
+    }
+}
+
+impl<'a> FromLH<'a, U256> for Binary<'a> {
+    fn from(value: U256, env: rustler::Env<'a>) -> Self {
+        bytes_to_binary(env, &value.as_ssz_bytes())
+    }
+}
+
+impl<'a> FromLH<'a, U256> for u64 {
+    fn from(value: U256, _env: rustler::Env<'a>) -> Self {
+        u64::try_from(value).unwrap()
+    }
+}
+
+impl<'a> FromLH<'a, VariableList<u8, <MainnetEthSpec as EthSpec>::MaxExtraDataBytes>>
+    for Binary<'a>
+{
+    fn from(
+        value: VariableList<u8, <MainnetEthSpec as EthSpec>::MaxExtraDataBytes>,
+        env: rustler::Env<'a>,
+    ) -> Self {
+        bytes_to_binary(env, &value.as_ssz_bytes())
+    }
+}
+
+impl<'a> FromLH<'a, VariableList<u8, <MainnetEthSpec as EthSpec>::MaxExtraDataBytes>> for Vec<u8> {
```

Comment:
We can use the `FromLH<'a, FixedVector<Lh, N>> for Vec<Elx>` implementation as a base.

Reply:
Same a previously we need to represent a Vec<> -> Transactions, of Vec<u8> which are a transactions